artilleryScripts.js

```diff
@@ -0,0 +1,5 @@
+module.exports.randomId = (userContext, ee, next) => {

```

Comment:
A very minor point, but it's odd to have this file in the pull request when all it has in it is a function you don't use anymore.

Reply:
yeah, maybe not the best to put this in, but I will need the file when I set things up for benchmarking push requests, so I didn't want to remove it entirely